src/sendqueue.rs

```diff
@@ -14,11 +15,31 @@ use crate::{Active, Capture};
 
 pub struct SendQueue(NonNull<raw::pcap_send_queue>);
 
-pub enum Sync {
+pub enum SendSync {
     Off = 0,
     On = 1,
 }
 
+#[inline]
+fn mkpkthdr(ts: Option<std::time::Duration>, len: u32) -> raw::pcap_pkthdr {
```

Comment:
In case this function ends up being reused outside of sendqueue, can you please rename it to `make_pkthdr` for consistent naming with other functions.